### DISCUSSION TITLE
Dall-E-3 Support

### DISCUSSION THREAD

**Discussion by BarGuy** [knrd1/chatgpt#36] [Ideas]
Please add the ability to enable dall-e-3 support

**Comment by cleobuline** [comment]
replace "dalle" in the script and config   by "dall-e-3"

**Comment by BarGuy** [comment]
Tried that. I get invalid model. Guess my plan doesn’t cover dalle3TKeep Calm & Chive On!!!On Feb 21, 2024, at 17:24, cleobuline ***@***.***> wrote:﻿
replace "dalle" in the script and config   by "dall-e-3"

**Comment by BarGuy** [comment]
Actually I think I should be able to use dalle3. I can use dalle Keep Calm & Chive On!!!On Feb 22, 2024, at 09:16, Dan Stark ***@***.***> wrote:﻿Tried that. I get invalid model. Guess my plan doesn’t cover dalle3TKeep Calm & Chive On!!!On Feb 21, 2024, at 17:24, cleobuline ***@***.***> wrote:﻿
replace "dalle" in the script and config   by "dall-e-3"

**Comment by knrd1** [comment]
Hey, thanks for your patience, I just added dall-e-2 and dall-e-3 support!
